internal/core/command/get.go

```diff
@@ -49,5 +63,11 @@ func NewGetCommand(
 		request.Header.Set(clients.CorrelationHeader, correlationID.(string))
 	}
 
+	addHeadersToRequest(request, r.Header.Get(clients.ContentType))
+
 	return newServiceCommand(device, httpCaller, request, lc), nil
 }
+
+func addHeadersToRequest(r *http.Request, value string) {
```

Comment:
The idea for my previous comment in favor of this helper function was so that this function would encapsulate all the business logic within a single place which handles all the headers that need to be added to the out going request. Could we make this function add `Content-type` and `Correlation`? If so, like my previous comment about this helper function, I think we should do this in a smart way which allows us to extend functionality within requiring modification. The reason I say this is because we very well may want to add or remove headers from the "proxied" request in the future. Therefore, I would like to do this a little smarter and avoid hard-coding which headers are being propagated. See my previous comment from a previous iteration for more details and a starting idea as to how to accomplish this.